packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -153,9 +159,8 @@ const Combobox = React.forwardRef<HTMLInputElement, ComboboxProps>(
         >
           <ComboboxInputWrapper {...getComboboxProps({role: 'combobox'})}>
             <ComboboxInputSelect
-              {...getToggleButtonProps({tabIndex: 0})}
               // we spread props into `getInputProps` so that Downshift handles events correctly
-              {...getInputProps({disabled, required, ref, ...props})}
+              {...getInputProps({disabled, required, ...props, ...toggleButtonProps})}
```

Comment:
Is it safe to pass toggleButtonProps into getInputProps? (Made a similar comment elsewhere, pointing out both locations)

Reply:
I [left a response to this right here](https://github.com/twilio-labs/paste/pull/2155/files#r793115262) 👈